examples/features/metadata_interceptor/README.md

```diff
@@ -67,4 +67,4 @@ func SomeStreamInterceptor(srv interface{}, ss grpc.ServerStream, info *grpc.Str
 
     return handler(srv, &wrappedStream{ss, ctx})
 }
-```
\ No newline at end of file
+```
```

Comment:
new line should be after quotes before eof

Reply:
It is after the quotes. Its just how the difference shows up because there is nothing in newline.